_docs/whatiscloud.md

```diff
@@ -20,7 +20,7 @@ There are two kinds of offerings that you might see when browsing a cloud provid
 * Services, on the other hand, abstract away the nitty-gritty implementation details. Services allow you to build and deploy your application without worrying about the underlying details. For example, Amazon’s AWS Lambda service allows you to deploy functions that contain regular code. AWS might run your code in a VM, or in a Container. The key thing to remember here is, *it doesn’t matter to you*. Most cloud offerings fall into the Services category, and they make developing cloud applications much easier than it might appear.
 
 ### Intriguing...so what can I do in the cloud?
-Almost anything you can dream of. Cloud providers are constantly improving and adding to their offerings. A good place to start would be to take a simple application that you could run locally, such as an app that retrieves and stores information in a database and move it into the cloud. Refer to [LINK TO EXAMPLE PROJECTS] for a few concrete examples.
+Almost anything you can dream of. Cloud providers are constantly improving and adding to their offerings. A good place to start would be to take a simple application that you could run locally, such as an app that retrieves and stores information in a database and move it into the cloud. Refer to [our collection of example projects](project_ideas.html) for a few concrete examples.
```

Comment:
TIL that Jekyll has a great Liquid template for generating correct internal links. See [the docs](https://jekyllrb.com/docs/liquid/tags/#link) for syntax details. One of the nice things about this (depending on your opinion) is that broken links will cause the Jekyll build to fail, which is something I'm okay with in this instance. NOTE: this will be a big fat test of this functionality.

```suggestion
Almost anything you can dream of. Cloud providers are constantly improving and adding to their offerings. A good place to start would be to take a simple application that you could run locally, such as an app that retrieves and stores information in a database and move it into the cloud. Refer to [our collection of example projects]({% link _docs/project-ideas.md %}) for a few concrete examples.
```